.env-example

```diff
@@ -44,7 +44,9 @@ export POSTMAN_API_KEY=""
 export POSTMAN_SMS_CRED_NAME=""
 
 # Amplify
-AWS_REGION="ap-southeast-1"
+export AWS_REGION="ap-southeast-1"
+export AWS_ACCOUNT_NUMBER="000000000000"
+export IS_SITE_LAUNCH_DEV=true
```

Comment:
We should use the existing `NODE_DEV` env var for this

Reply:
[4abf7bb](https://github.com/isomerpages/isomercms-backend/pull/538/commits/4abf7bbf969779ded606fdbacaff4e6fe62bb08a)